Amplitude-iOS.podspec

```diff
@@ -6,8 +6,9 @@ Pod::Spec.new do |s|
   s.license                = { :type => "MIT" }
   s.author                 = { "Amplitude" => "dev@amplitude.com" }
   s.source                 = { :git => "https://github.com/amplitude/Amplitude-iOS.git", :tag => "v4.10.0" }
-  s.ios.deployment_target  = '8.0'
+  s.ios.deployment_target  = '13.2'
   s.tvos.deployment_target = '9.0'
+  s.osx.deployment_target  = '10.15'
```

Comment:
Does this mean that the minimum OS X deployment target is 10.15? Can we make it to be 10.10?